rewrite-java/src/main/java/org/openrewrite/java/internal/template/JavaTemplateLanguageExtension.java

```diff
@@ -40,8 +40,8 @@ public abstract class JavaTemplateLanguageExtension {
     Space.Location loc;
     JavaCoordinates.Mode mode;
 
-    public JavaTemplateLanguageExtension(JavaTemplateParser templateParser, Substitutions substitutions,
-                                         String substitutedTemplate, JavaCoordinates coordinates) {
+    protected JavaTemplateLanguageExtension(JavaTemplateParser templateParser, Substitutions substitutions,
```

Comment:
Do not reduce this visibility. This is used in parsers for other languages not yet merged into the main rewrite repository.